Mathlib/Topology/UniformSpace/Basic.lean

```diff
@@ -1120,6 +1120,15 @@ nonrec theorem UniformContinuous.comp [UniformSpace β] [UniformSpace γ] {g : 
   hg.comp hf
 #align uniform_continuous.comp UniformContinuous.comp
 
+/--If a function `T` is uniformly continuous in a uniform space `β`,
+then its `n`-th iterate `T^[n]` is also uniformly continuous.-/
+theorem uniformContinuous_ite [UniformSpace β] (T : β → β) (n : ℕ) (h : UniformContinuous T) :
+    UniformContinuous T^[n] := by
+  induction' n with n hn
```

Comment:
Does `induction` work instead of `induction'`? (It is more canonical).

Reply:
No, it doesn't work if I naively substitute it for `induction'`.

Reply:
Sure, naive substitution won't work, but `induction'` is suboptimal and will probably be removed some day, so better learn the new syntax. Can you figure it out (for instance looking for `induction n` around)? If you can't, I can show you how it should be done, but I think it's more instructive to let you find out.

Reply:
Sure, I'll do it as soon as I can.

Reply:
Done.